services/ui-src/src/components/app/App.tsx

```diff
@@ -1,12 +1,15 @@
+import { useContext } from "react";
 import { Routes, Route } from "react-router-dom";
-import { LoginCognito, LoginIDM, PostLogoutRedirect } from "components";
-import { useStore } from "utils";
-import { makeMediaQueryClasses } from "utils/other/useBreakpoint";
-import { Container, Divider, Heading, Stack } from "@chakra-ui/react";
+// components
```

Comment:
tiny nit: please remove these style comments in hcbs, we don't need to track the order here